mlflow/store/artifact/utils/models.py

```diff
@@ -63,28 +57,31 @@ def _parse_model_uri(uri):
     if parsed.scheme != "models":
         raise MlflowException(_improper_model_uri_msg(uri))
     path = parsed.path
-    m = _MODEL_URI_REGEX.match(path)
-    if m is None:
+    if not path.startswith("/") or len(path) <= 1:
         raise MlflowException(_improper_model_uri_msg(uri))
-    gd = m.groupdict()
-    model_name = gd.get("model_name")
-    suffix = gd.get("suffix")
-    alias = gd.get("alias")
-    if (model_name.strip() == "") or (suffix and alias) or (suffix is None and alias is None):
+
+    parts = path[1:].split("/")
+    if len(parts) < 1 or len(parts) > 2 or parts[0].strip() == "":
```

Comment:
do we need the first check here? If the len(parts) == 0, the above check would throw, right?